galaxy_ng/app/api/ui/viewsets/execution_environment.py

```diff
@@ -247,11 +247,15 @@ def get_object(self):
 class ContainerRegistryRemoteFilter(filterset.FilterSet):
     name = filters.CharFilter(field_name='name')
     url = filters.CharFilter(field_name='url')
+    pulp_created = filters.CharFilter(field_name='pulp_created')
```

Comment:
@newswangerd do we want to name the public field `pulp_created` ? or only the `field_name=` argument value?

Reply:
@newswangerd let me know re: Bruno's question.

Reply:
The public names should be `created_at` and `updated_at`

I also don't think you need to add these pulp_created and updated filters here. They're date fields in the database, not charfields, so this search isn't going to work anyway.

Reply:
@newswangerd tried to remove the lines corresponding to 250/251 and no-go, didn't work in testing...